### DISCUSSION TITLE
[Diffusion in FP8] - Where is this?

### DISCUSSION THREAD

**Discussion by Edu2014** [lllyasviel/stable-diffusion-webui-forge#1005] [General]
![image](https://github.com/user-attachments/assets/bbb0b6b8-9db0-4b18-ae5f-b2a91511d51a)

How to make it appear? I tried to activate it in the quicksettings list, but I couldn't find it. And it's not on the menu either.

**Comment by Edu2014** [comment]
![image](https://github.com/user-attachments/assets/9e4880c7-27a3-497f-aa6e-39176b9ddc55)

I believe it's the same thing

**Comment by danilomaiaweb** [comment]
> ![image](https://private-user-images.githubusercontent.com/8563692/356909697-bbb0b6b8-9db0-4b18-ae5f-b2a91511d51a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM3NTA4MDQsIm5iZiI6MTcyMzc1MDUwNCwicGF0aCI6Ii84NTYzNjkyLzM1NjkwOTY5Ny1iYmIwYjZiOC05ZGIwLTRiMTgtYWU1Zi1iMmE5MTUxMWQ1MWEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgxNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MTVUMTkzNTA0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGNmZGY2Y2ViMTUwYjI1OWViODNhNGEwMjE4MGVlOTFhNDQxOWIyYzkyNmI5ZDM5ZTUzZmE2NTg0MTliNTg5NCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.vFBwEXL1nMX5RkO9vBbf2qts5upvhO3qZCd1cV_s5xg)
> 
> How to make it appear? I tried to activate it in the quicksettings list, but I couldn't find it. And it's not on the menu either.



Settings > User interface > digite fp8 on second input
Apply settings
Restart

![image](https://github.com/user-attachments/assets/86087e23-6c60-4196-b014-7a40d3355128)

How the option show on top with other settings:
![image](https://github.com/user-attachments/assets/bf46e465-a514-4b74-839d-d6ebe3bfb94b)

**Reply by Edu2014** [reply]
![image](https://github.com/user-attachments/assets/375f9dd0-1483-4699-b44f-768ef31b2392)

I meant to say, you can use model 1.5 with bnb-nf4,bnb-fp4,float8-e5m2
